src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -134,9 +134,6 @@ def __init__(self, port = '/dev/ttyUSB0', baudRate = 57600, address = 0xFFFFFFFF
         @param integer(4 bytes) password
         """
 
-        if ( os.path.exists(port) == False ):
```

Comment:
Is there no possibility on windows to check if the given port is accessible? I really don't feed comfortable to simply remove this check. Alternatively we could check it only for non-Windows.

Reply:
An exception is raised by PySerial if the port does not exist! I think that in this case the underlying PySerial library should take care of this. Anyway we could use `serial.tools.list_ports.comports` (https://pyserial.readthedocs.io/en/latest/tools.html#module-serial.tools.list_ports) to get a list of available ports and check against the given one.

Reply:
Okay